sentinel-dashboard/pom.xml

```diff
@@ -13,6 +13,7 @@
     <packaging>jar</packaging>
 
     <properties>
+        <resource.delimiter>@</resource.delimiter>
```

Comment:
Do we need to specify the delimiter explicitly? 
It seems that the default delimiter is `@`, and the `maven-resources-plugin` can also be omitted.
Certainly, explicit specification has the effect of emphasis. 
Using default or specify the delimiter and resource plugin, which do you think is better?

Reply:
I'm quite sorry, the project does not inherit spring boot starter parent.
Specify the delimiter is right and adding `maven-resources-plugin` is necessary.